cardinal/plugin_debug.go

```diff
@@ -48,10 +48,10 @@ func (p *debugPlugin) RegisterQueries(world *World) error {
 
 // queryDebugState godoc
 //
-//	@Summary		Get information on all entities and components in world-engine
-//	@Description	Displays the entire game state.
+//	@Summary		Get all game entities
```

Comment:
nit: All these indentations are wonky. gofmt uses spaces to align text horizontally, however that tool isn't run inside of comments.

Can you update these indentations to use spaces instead of tabs?